plugins/shorten_urls.lua

```diff
@@ -74,8 +74,9 @@ return {
 					end
 				end
 				if #msg > 0 then
-					msg = sender[1] .. ": " .. msg
-					irc:PRIVMSG(origin, msg)
+					nick = sender[1]:gsub("(.)(.+)", "%1​%2")
```

Comment:
How do you include the no-width-space here without another module, while also making it visible in all editors?

GitHub doesn't display the no-width space for me